pkg/cmd/pulumi/package.go

```diff
@@ -17,6 +17,7 @@ package main
 import (
 	"encoding/json"
 	"fmt"
+	"github.com/pulumi/pulumi/sdk/v3/go/common/env"
```

Comment:
If you don't have gofumpt installed, you can probably also run `golangci-lint run --fix` — I think.
If you use VSCode, you can follow the instructions at https://github.com/mvdan/gofumpt#visual-studio-code to configure it to use gofumpt. Use workspace-level settings instead of global settings to make this change only for pu/pu. (We have this enabled in the .vscode in pu/pu's root, but that won't work if you create a project in pkg/ and cmd/ separately, which is likely something you're doing because otherwise gopls doesn't work quite right in VSCode yet.)

Reply:
If `gofumpt` is the preferred linter for this team, could this process be documented somewhere besides an offhand review comment?

And Goland users can use File Watcher to run this on every file save, lets try to be inclusive of more developer environments 😁

Reply:
We mention gofumpt in the CONTRIBUTING.md under [setting up your environment](https://github.com/pulumi/pulumi/blob/master/CONTRIBUTING.md#setting-up-your-pulumi-development-environment), although I suspect you already had a working pu/pu environment—the gofumpt change was made only a month ago.

Reply:
@jazzyfresh There was room for improvement in the relevant section of the CONTRIBUTING.md.
Updated: https://github.com/pulumi/pulumi/pull/12860